README.md

```diff
@@ -44,10 +44,6 @@ func main() {
 }
 ```
 
-Documentation
```

Comment:
suggestion: this should be updated to point to thr new docs site instead of removed